Content.Server/GameObjects/Components/Movement/AiControllerComponent.cs

```diff
@@ -61,7 +61,7 @@ public override void ExposeData(ObjectSerializer serializer)
         protected override void Shutdown()
         {
             base.Shutdown();
-            Processor.Shutdown();
+            Processor?.Shutdown();
```

Comment:
Can anyone tell me why this is necessary now? It's _fine_ but I don't know why.

Reply:
The AiSystem initializes the processor so if it's not getting run in the tests that could cause it. I'd need to look further into it.